e2e-tests/src/test/staking.rs

```diff
@@ -198,7 +198,9 @@ pub async fn staking_new_validator() -> anyhow::Result<()> {
             total: MIN_VALIDATOR_BOND,
             active: MIN_VALIDATOR_BOND,
             unlocking: BoundedVec(vec![]),
-            claimed_rewards: BoundedVec(vec![]),
+            // since era is 3 sessions, validate is done in the first block of 2nd session,
+            // that is already after elections has been done for 1st era
+            claimed_rewards: BoundedVec(vec![0]),
```

Comment:
I need another explanation why this `0` is needed :P this one didn't help

Reply:
Testcase looks as follows, narrowed to important to out discssion
* 1st block -> issue `elections.changeValidators` with 4 `non_reserved` nodse
* wait 2 sessions
* 60 th block -> issue `staking.validate` 

Now before this PR, in 60th block there was already session 0 (active era == current era == 0), but now there's already session 1 to some extent (active era == 1, current era == 0), as in block 59th elections has happened. Hence, when `staking.validate` is done, pallet staking adds to `ledger.claimedRewards` all past eras till active era -1. This is a known mechanism in pallet staking to prevent claiming past eras.